heareval/tasks/util/luigi.py

```diff
@@ -121,7 +121,9 @@ def download_file(url, local_filename, expected_md5):
                 f.write(chunk)
                 pbar.update(chunk_size)
             pbar.close()
-    assert md5sum(local_filename) == expected_md5
+    # If expected_md5 is 0, the check is not done
+    if expected_md5:
+        assert md5sum(local_filename) == expected_md5
```

Comment:
If md5 is an empty string, the md5 will not be checked. Since the dataset for downloading is small enough in small tasks we are not checking the md5.

Reply:
Can we still use the small file's md5sum ?? then we can verify everything is correct

any way we can easily check also that the small file has a certain # of instances and certain # of labels in the final postprocessed version?

Reply:
Changed to using the md5 for the small dataset as well